src/gc/gc.cpp

```diff
@@ -1727,12 +1727,6 @@ static void enter_spin_lock_noinstru (RAW_KEYWORD(volatile) int32_t* lock)
     }
 }
 
-inline
-static BOOL try_enter_spin_lock_noinstru(RAW_KEYWORD(volatile) int32_t* lock)
```

Comment:
cc: @Maoni0

Reply:
Removed this change and added Wno-unused-function instead as more warnings like this popped up.